### PR TITLE
Final graph update

### PR DESCRIPTION
**Pull request by beckskelly**
Final PR (for real) as graph looked a bit bad live for portfolios

### CODE REVIEW

File: src/components/gameComponents/individualGameScreenComponents/gameNavigation/GamePortfolio.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ function GamePortfolio({ data, name, totalValue }) {
         return lineColor
     }
 
+    console.log(data)
```
Comment (BearachB) [inline src/components/gameComponents/individualGameScreenComponents/gameNavigation/GamePortfolio.js:30]: Might want to remove this?

**Review by BearachB — CHANGES_REQUESTED** [review]
Looks really good @beckskelly. Really good work on this. Scales really well now, very responsive. Dates are much easier to see. Also the ticks on mobile are much easier to see now.

Just a tiny change needed, just spotted a console log in GamePortfolio.js

File: src/components/gameComponents/individualGameScreenComponents/gameNavigation/GamePortfolio.js (hunk 1 above)
Comment (beckskelly) [reply src/components/gameComponents/individualGameScreenComponents/gameNavigation/GamePortfolio.js]: done, thank you!

**Review by BearachB — APPROVED** [review]
Thanks for that update @beckskelly! Looks good to me however someone who's more familiar with the graphs might find a problem?

**Review by DaraghK93 — APPROVED** [review]
looks good Becks, cheers

**Pull request merged**